src/calendar-list/item.js

```diff
@@ -44,6 +44,9 @@ class CalendarListItem extends Component {
           dayComponent={this.props.dayComponent}
           disabledByDefault={this.props.disabledByDefault}
           showWeekNumbers={this.props.showWeekNumbers}
+          renderArrow={this.props.renderArrow === undefined ? undefined : this.props.renderArrow}
```

Comment:
The check for undefined is redundant. Just pass the prop as is

Reply:
Of course !